src/reconstruction/refinement/addMultipleMetabolites.m

```diff
@@ -0,0 +1,79 @@
+function newmodel = addMultipleMetabolites(model,metIDs,varargin)
+%Add a batch of metabolites to the model.
```

Comment:
May you give here a more explicit explanation? Also, maybe refer to `addMetabolite`.